### ISSUE TITLE
TalkBack crashing while calling GMS OCR features

### ISSUE THREAD

**Issue by falisca** [GrapheneOS/os-issue-tracker#2159]
Hi, I'm having TalkBack crashing frequently and I'm not sure what's causing it.
Note: tested on Pixel 6A and on  a (legacy) Pixel 3a with the latest OS updates from the stable channel on both.
I've been having these crashes for a couple of months now, I'm guessing some recent work on the GSF sandbox or an unattended TalkBack update broke something.
I've noticed the crashes occur when I hover TalkBack focus on every unlabeled button in the apps UI and also in  some default Graphene apps (especially the clock app in the alarm clock tab).
Inspecting the logs I can tell  that the feature of TalkBack that perform OCR on UI elements, with images or controls without the proper A11Y labels, fails while calling GMS  OCR features:
the TalkBack service tryes to call frequently the  GMS API, especially a "MlkitOcrCommon" library that can't be found.
Due to the log entries the library cannot be opened, I wonder if this is because the GSF sandbox is blocking this call.
Note that after two crashes TalkBack never resumes, forcing to reboot the device. The OS is practically unusable with third party apps.
### Steps to reproduce:
1. Enable TalkBack service.
2. Install a TTS if you don't have it and choose it in TalkBack voices settings.
3. Open the Clock app and switch on the alarm tab.
4. Use the flick gesture (swipe a finger left and or right) to navigate the UI.
### Third party app to try instead: Antennapod
1. Repeat steps 1 and 2 from the steps above.
2. Open Antennapod and from the menu choose one feed you are subscribed

Could anyone confirm this behavior? Any ideas on the logs? Could be fixable?
[Pixel3A_TalkBack_crash.log](https://github.com/GrapheneOS/os-issue-tracker/files/11360934/Pixel3A_TalkBack_crash.log)
[Pixel6A_TalkBack_crash.log](https://github.com/GrapheneOS/os-issue-tracker/files/11360935/Pixel6A_TalkBack_crash.log)

**Comment by gcocevar** [comment]
I have the same problem.
I see that in the 2023041100 release notes from the stable channel, TalkBack was mentioned: "(screen reader): update base code to 13.0 and overhaul our changes for it including removing proprietary library dependency". Maybe the proprietary library dependencyes weren't completely removed  or something went wrong?

**Comment by falisca** [comment]
No longer applicable.

**Comment by falisca** [comment]
No longer applicable
